language_names/build_db.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import sqlite3
+import textwrap
+
+
+def insert_values(c, filename, tablename):
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS {} (
+            id INTEGER PRIMARY KEY,
+            lg TEXT,
+            inLg TEXT,
+            name TEXT,
+            UNIQUE(lg, inLg) ON CONFLICT IGNORE);
+        """.format(tablename)))
+    with open(filename, 'r') as f:
+        reader = csv.DictReader(f, delimiter='\t')
+        for row in reader:
+            c.execute('INSERT INTO {} VALUES (?, ?, ?, ?)'.format(tablename), (None, row['lg'], row['inLg'], row['name']))
+
+
+def populate_database():
+    conn = sqlite3.connect('langNames.db')
```

Comment:
This should use the argument.

Reply:
Ohh now I understand why you wanted me to pass langNames.db